### ISSUE TITLE
Line causes log warning and errors due to "blocking call" #663

### ISSUE THREAD

**Issue by Tozapid** [jekalmin/extended_openai_conversation#258]
As far as i can see this issue has not been reported before

> please create a bug report at https://github.com/jekalmin/extended_openai_conversation/issues For developers, please see https://developers.home-assistant.io/docs/asyncio_blocking_operations/#load_verify_locations 

```
Core 2024.10.2
Supervisor 2024.10.2
Operating System 13.2
Frontend 20241002.3
```

```
Logger: homeassistant.util.loop
Source: util/loop.py:136
First occurred: 1:12:59 AM (4 occurrences)
Last logged: 1:13:01 AM

Detected blocking call to load_verify_locations with args (<ssl.SSLContext object at 0x7f6863a38550>,) inside the event loop by custom integration 'extended_openai_conversation' at custom_components/extended_openai_conversation/helpers.py, line 146: client = AsyncOpenAI( (offender: /usr/local/lib/python3.12/site-packages/httpx/_config.py, line 149: context.load_verify_locations(cafile=cafile)), please create a bug report at https://github.com/jekalmin/extended_openai_conversation/issues For developers, please see https://developers.home-assistant.io/docs/asyncio_blocking_operations/#load_verify_locations Traceback (most recent call last): File "<frozen runpy>", line 198, in _run_module_as_main File "<frozen runpy>", line 88, in _run_code File "/usr/src/homeassistant/homeassistant/__main__.py", line 223, in <module> sys.exit(main()) File "/usr/src/homeassistant/homeassistant/__main__.py", line 209, in main exit_code = runner.run(runtime_conf) File "/usr/src/homeassistant/homeassistant/runner.py", line 189, in run return loop.run_until_complete(setup_and_run_hass(runtime_config)) File "/usr/local/lib/python3.12/asyncio/base_events.py", line 674, in run_until_complete self.run_forever() File "/usr/local/lib/python3.12/asyncio/base_events.py", line 641, in run_forever self._run_once() File "/usr/local/lib/python3.12/asyncio/base_events.py", line 1990, in _run_once handle._run() File "/usr/local/lib/python3.12/asyncio/events.py", line 88, in _run self._context.run(self._callback, *self._args) File "/usr/src/homeassistant/homeassistant/setup.py", line 165, in async_setup_component result = await _async_setup_component(hass, domain, config) File "/usr/src/homeassistant/homeassistant/setup.py", line 461, in _async_setup_component await asyncio.gather( File "/usr/src/homeassistant/homeassistant/setup.py", line 463, in <genexpr> create_eager_task( File "/usr/src/homeassistant/homeassistant/util/async_.py", line 45, in create_eager_task return Task(coro, loop=loop, name=name, eager_start=True) File "/usr/src/homeassistant/homeassistant/config_entries.py", line 742, in async_setup_locked await self.async_setup(hass, integration=integration) File "/usr/src/homeassistant/homeassistant/config_entries.py", line 594, in async_setup result = await component.async_setup_entry(hass, self) File "/config/custom_components/extended_openai_conversation/__init__.py", line 100, in async_setup_entry await validate_authentication( File "/config/custom_components/extended_openai_conversation/helpers.py", line 146, in validate_authentication client = AsyncOpenAI(
```

**Comment by Tozapid** [comment]
I noticed similar issues in other extensions that have already added pull requests, maybe these links will help

* https://github.com/humbertogontijo/homeassistant-roborock/issues/663
* https://github.com/home-assistant/core/issues/124942
* https://github.com/alandtse/tesla/issues/1047

**Comment by jekalmin** [comment]
Thanks for reporting an issue!
It's released in [1.0.4](https://github.com/jekalmin/extended_openai_conversation/releases/tag/1.0.4)

Please reopen it if problem still exists.
